lib/cequel/metal/keyspace.rb

```diff
@@ -245,6 +248,13 @@ def default_consistency
         @default_consistency || :quorum
       end
 
+      #
+      # @return [Symbol] the client compression setting
+      #
+      def client_compression
+        @client_compression
+      end
```

Comment:
To be honest, I don't think so. I added this to follow convention in keyspace.rb where other configuration options did the same.